bbb-install.sh

```diff
@@ -123,9 +124,13 @@ main() {
         exit 0
         ;;
 
+      p)
+        echo "Acquire::http::Proxy \"http://$OPTARG:3142\";"  > /etc/apt/apt.conf.d/01proxy
+        ;;
```

Comment:
Moved the proxy check before the host check, because in the host section will be `check_host` called which calls `need_pkg` which calls `apt-get install`

Reply:
Oh, stupid me. The order depends on how it is in the arguments and not on the order in which they are written here.

Then the script still has the problem that when using an apt-get proxy, apt-get commands may be executed before the proxy is effective.